client/dashboard/me/monetize-subscriptions/monetize-subscription.tsx

```diff
@@ -0,0 +1,307 @@
+										actions: [
+											{
+												url: CALYPSO_CONTACT,
+												label: __( 'Please contact support' ),
+											},
+										],
+									} );
+								} else {
+									createErrorNotice(
+										__( 'There was a problem while stopping your subscription.' ),
+										{
+											actions: [
+												{
+													url: CALYPSO_CONTACT,
+													label: __( 'Please contact support' ),
+												},
+											],
+										}
+									);
+								}
+							},
+						} );
+					} }
+				>
+					{ title }
+				</Button>
+			}
+		/>
+	);
+}
+
+const BackButton = () => {
+	const router = useRouter();
+
+	return (
+		<Button
+			className="dashboard-page-header__back-button"
+			icon={ isRTL() ? chevronRight : chevronLeft }
+			onClick={ () => {
+				router.navigate( {
+					to: getMonetizeSubscriptionsUrl(),
+				} );
+			} }
+		>
+			{ getMonetizeSubscriptionsPageTitle() }
+		</Button>
+	);
+};
+
+export default function MonetizeSubscription() {
+	const params = monetizeSubscriptionRoute.useParams();
+	const subscriptionId: string = params.subscriptionId;
+
+	const { data: subscription } = useQuery( monetizeSubscriptionQuery( subscriptionId ) );
+
+	const { mutate: stopSubscription, isPending: isStoppingSubscription } = useMutation(
+		monetizeSubscriptionStop( subscriptionId )
+	);
+
+	const { mutate: disableAutoRenew, isPending: isDisablingAutoRenew } = useMutation(
+		monetizeSubscriptionDisableAutoRenew( subscriptionId )
+	);
+
+	const { mutate: enableAutoRenew, isPending: isEnablingAutoRenew } = useMutation(
+		monetizeSubscriptionResumeAutoRenew( subscriptionId )
+	);
+
+	const isRenewable = subscription && ( subscription.renew_interval || subscription.is_renewable ); // can remove renew_interval once backend is deployed
+	const isAutoRenewing = isRenewable && subscription.renew_interval;
+	const isProduct = !! subscription && ! isRenewable;
+	const isDisabledAutorenewing = isRenewable && ! subscription.renew_interval;
+	const isUpdating = isEnablingAutoRenew || isDisablingAutoRenew;
+
+	return (
+		<PageLayout
+			size="small"
+			header={
+				<PageHeader
+					prefix={ <BackButton /> }
+					title={ isProduct ? __( 'Product Details' ) : __( 'Subscription Details' ) }
+				/>
+			}
+		>
+			{ isStoppingSubscription && (
+				<Notice status="info">
+					{ isProduct ? __( 'Removing this product' ) : __( 'Stopping this subscription' ) }
+				</Notice>
+			) }
+			{ isUpdating && <Notice status="info">{ __( 'Updating subscription auto-renew' ) }</Notice> }
+			{ subscription && (
+				<Card>
+					<MonetizeSiteHeader name={ subscription.site_title } domain={ subscription.site_url } />
+
+					<CardHeader>
+						<HStack alignment="center">
+							<VStack>
+								<HStack alignment="center">
+									<h3>{ subscription.title }</h3>
+									<b>
+										{ formatCurrency(
+											parseFloat( subscription.renewal_price ),
+											subscription.currency
+										) }
+									</b>
```

Comment:
nit: sames as [here](https://github.com/Automattic/wp-calypso/pull/105695/files#r2376503732).


```suggestion
									<Text as="h3">{ subscription.title }</Text>
									<Text weight="700">
										{ formatCurrency(
											parseFloat( subscription.renewal_price ),
											subscription.currency
										) }
									</Text>
```

Reply:
Done in 15fdc7c17f9b1a5e91acb9ab4f13ec029d131641. I actually used size="title"...